src/utils.ts

```diff
@@ -25,15 +25,24 @@ export async function setForkBlock(blockNumber: number) {
 
 export function getCalldatas({ signatures, params }: { signatures: string[]; params: any[][] }) {
   return params.map((args: any[], i: number) => {
-    const types = getArgs(signatures[i]);
+    let types = getArgs(signatures[i]);
+    // Fix for the oracle VIP as there is struct in types and defaultAbiCoder
+    // is unable to process struct.
+
+    if (signatures[i] == "setTokenConfig((address,address,uint256))") {
+      types = ["tuple(address, address, uint256)"];
+    } else if (signatures[i] == "setTokenConfig((address,address[3],bool[3]))") {
+      types = ["tuple(address, address[3], bool[3])"];
+    }
```

Comment:
Would it make sense to try to do something generic with a regexp to capture the structs? We may keep this at it is now, and review it in the following days...